### PR TITLE
Konstantin/appeals 52321

### PR DESCRIPTION
**Pull request by KonstantinShevtsov**
Resolves [APPEALS-52321](https://jira.devops.va.gov/browse/APPEALS-52321)

# Description
That PR adds a method in RequestIssueSerializer that converts rating_profile_date to UTC format and 6 unit tests for it. Also during work on that PR performed research on contested issues rating_profile_date and its relations, (see diagram in Jira story) 

## Acceptance Criteria
- [x] Code compiles correctly

## Testing Plan
N/A

### Test Coverage
Did you include any test coverage for your code? Check below:
- [x] RSpec

### Code Climate
Your code does not add any new code climate offenses? If so why?
- [x] No new code climate issues added

### Monitoring
- [x] Are performance metrics (e.g., response time, throughput) being tracked?
- [x] Are key application components monitored (e.g., database, cache, queues)?
- [x] Is there a system in place for setting up alerts based on performance thresholds?

### CODE REVIEW

File: app/serializers/api/v3/issues/ama/request_issue_serializer.rb [hunk 1]
```diff
@@ -137,4 +138,11 @@ class Api::V3::Issues::Ama::RequestIssueSerializer
       object&.end_product_establishment&.user&.station_id
     end
   end
+
+  def self.format_rating_profile_date(date)
```
Comment (nkutub) [inline app/serializers/api/v3/issues/ama/request_issue_serializer.rb:142]: The date will always be string in this field? also what about if the parse of the date fails ?

Comment (nkutub) [reply app/serializers/api/v3/issues/ama/request_issue_serializer.rb:142]: you need to test for a badly formated date and just return the string if it cant be parsed

Comment (KonstantinShevtsov) [reply app/serializers/api/v3/issues/ama/request_issue_serializer.rb:142]: > The date will always be string in this field? also what about if the parse of the date fails ?

Good catch, thank you. Will fix it immediately.

Comment (KonstantinShevtsov) [reply app/serializers/api/v3/issues/ama/request_issue_serializer.rb:142]: > The date will always be string in this field? also what about if the parse of the date fails ?

Oh, actually it converts DateTime to UTC not string only, date.utc at the bottom of the method. But you are right - I have to add something to handle invalid data

**Review by nkutub — APPROVED** [review]

**Pull request merged**